frontend/src/ts/menu.ts

```
@@ -53,12 +147,23 @@
 function initMenu(
   nextImageButton: HTMLElement,
   prevImageButton: HTMLElement,
+  showMoreInfo: boolean,
+  handleInfoKeyPress: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)